sycl/test-e2e/Basic/max_registers_per_work_group_query.cpp

```diff
@@ -0,0 +1,24 @@
+// REQUIRES: cuda || hip
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
Please change to using the new testing format, i.e. something like:

```suggestion
// RUN: %{build} -o %t.out
// RUN: %{run} %t.out
```